src/components/Navbar.jsx

```diff
@@ -1,20 +1,27 @@
 import React, { useState } from "react";
 import { Link, NavLink } from "react-router-dom";
-import "./navbar.css";
+
+import "./Navbar.css";
 import logo from "../Assets/logo.jpeg";
 import x from "../Assets/x-mark-24.png";
+import { useAuth } from './AuthContext';
 
 export const Navbar = () => {
   const [menuOpen, setMenuOpen] = useState(false);
+
+  // Define a variable to store login status. If logged in, display all
+  // navigation link, else display only about, login links.
+  const {isLoggedIn} = useAuth();
+  console.log('navbar element', isLoggedIn);//Debugging
```

Comment:
refrain from putting debugging residues into the pull request, but since this is still early in the development process it's fine :laughing: